CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 * [FEATURE] Add option to use jump hashing to load balance requests to memcached #1554
 * [FEATURE] Add status page for HA tracker to distributors #1546
+* [CHANGE] Added check for retention period to be a multiple of periodic table duration #1564
```

Comment:
This is user-facing so something like `Retention period should now be a multiple of periodic table duration` should better.